### PR TITLE
Update ZAP page

### PR DESCRIPTION
**Pull request by psiinon**

### CODE REVIEW

**Review by thc202 — APPROVED** [review]

File: index.md [hunk 1]
```diff
@@ -10,7 +10,8 @@ type: tool
 pitch: The OWASP Zed Attack Proxy (ZAP) is one of the world’s most popular free security tools and is actively maintained by a dedicated international team of volunteers. Great for pentesters, devs, QA, and CI/CD integration.
```
Comment (kingthorin) [inline index.md]:
```suggestion
pitch: Zed Attack Proxy (ZAP) is one of the world’s most popular free security tools and is actively maintained by a dedicated international team of volunteers. Great for pentesters, devs, QA, and CI/CD integration. 
```

Comment (kingthorin) [reply index.md]: Line 4 should drop the OWASP too.

**Comment by kingthorin** [comment]
Should we also do something with the "level" ?

**Comment by psiinon** [comment]
> Should we also do something with the "level" ?

I was going to check with Harold first - I think that can be done in another PR?

**Review by kingthorin — APPROVED** [review]

**Pull request merged**